.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Usually, you add something like this to a global git ignore, but adding it here won't hurt.